tests/ui/traits/new-solver/destruct.rs

```diff
@@ -0,0 +1,12 @@
+// check-pass
+
+#![feature(const_trait_impl)]
```

Comment:
Is this the feature that turns on the new trait solver?

Reply:
lmfao, no, I just forgot `// compile-flags: -Ztrait-solver=next`

Reply:
I guess that's why we do code review!